src/common/transformations/include/ngraph_ops/multiclass_nms_ie_internal.hpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <transformations_visibility.hpp>
+
+//#include "ngraph/coordinate_diff.hpp"
+#include "ngraph/opsets/opset9.hpp"
+
+namespace ngraph {
+namespace op {
+namespace internal {
+
+class TRANSFORMATIONS_API MulticlassNmsIEInternal : public opset9::MulticlassNms {
+public:
+    NGRAPH_RTTI_DECLARATION;
```

Comment:
Seems better to use the new style RTTI:
OPENVINO_OP("MulticlassNms", "ie_internal_opset");
BWDCMP_RTTI_DECLARATION;

Reply:
resolved